kettle/README.md

```diff
@@ -104,3 +104,7 @@ A [postsubmit job](https://github.com/kubernetes/test-infra/blob/master/config/j
 # Known Issues
 
 - Occasionally data from Kettle stops updating, we suspect this is due to a transient hang when contacting GCS ([#8800](https://github.com/kubernetes/test-infra/issues/8800)). If this happens, [restart kettle](#restarting)
+
+[Big Query Tables]: https://console.cloud.google.com/bigquery?utm_source=bqui&utm_medium=link&utm_campaign=classic&project=k8s-gubernator
+[Big Query All]: https://pantheon.corp.google.com/bigquery?project=k8s-gubernator&page=table&t=all&d=build&p=k8s-gubernator&redirect_from_classic=true
+[Big Query Staging]: https://pantheon.corp.google.com/bigquery?project=k8s-gubernator&page=table&t=staging&d=build&p=k8s-gubernator&redirect_from_classic=true
```

Comment:
```suggestion
[Big Query All]: https://console.cloud.google.com/bigquery?project=k8s-gubernator&page=table&t=all&d=build&p=k8s-gubernator&redirect_from_classic=true
[Big Query Staging]: https://console.cloud.google.com/bigquery?project=k8s-gubernator&page=table&t=staging&d=build&p=k8s-gubernator&redirect_from_classic=true
```

Reply:
dunno if the redirect_from_classic param needs to be there or not (I didn't get it using my @gmail account)

Reply:
Looks like it does not 👍

Reply:
Fixed with console.cloud changes also